policy/src/main/java/org/apache/brooklyn/policy/action/AbstractScheduledEffectorPolicy.java

```diff
@@ -101,15 +102,15 @@
 
     public static final ConfigKey<Boolean> RUNNING = ConfigKeys.builder(Boolean.class)
             .name("running")
-            .description("Set if the executor has started")
+            .description("[INTERNAL] Set if the executor has started")
             .defaultValue(Boolean.FALSE)
             .reconfigurable(true)
             .build();
 
     public static final ConfigKey<List<Long>> SCHEDULED = ConfigKeys.builder(new TypeToken<List<Long>>() { })
             .name("scheduled")
             .description("List of all scheduled execution start times")
-            .defaultValue(Lists.newCopyOnWriteArrayList())
+            .defaultValue(ImmutableList.of())
```

Comment:
We can use the non-COW version because you just re-set the config with a mutated copy, right?

Reply:
If you set the defaultValue to something mutable, then if two different policy instances retrieve and modify it, they'll see each other's changes!

And yes, because we're modifying a copy we can use the non-COW version.